docs/pages/x/api/date-pickers/next-date-picker.json

```diff
@@ -81,6 +82,12 @@
       }
     },
     "value": { "type": { "name": "any" } },
+    "view": {
```

Comment:
I have no idea how I missed it, but all the new pickers were missing the `view` prop